modules/local/staramr/main.nf

```diff
@@ -48,6 +48,9 @@ process STARAMR_SEARCH {
     # Add prefix ($meta.id) to the names of output files (allows for CSVTK module to concatenate files downstream)
     for f in ${prefix}_results/* ; do mv "\$f" \$(echo \$f | sed 's;/;/${prefix}_;'); done
 
+    # Add extension (.staramr) to the names of output files (to simplify output differntiation amongst other modules)
+    for f in ${prefix}_results/*{.tsv,.xlsx,.txt} ; do mv "\$f" \$(echo \$f | sed 's;\\.;\\.staramr\\.;'); done
```

Comment:
Thanks for this. The `sed` regex here won't work in a situation where a the prefix has a `.` in it (e.g., `SampleA.B.C`).

Could you switch the `sed` statement to something like below instead:

```
sed 's;\\.\\([^\\.]*\\)\$;\\.staramr\\.\\1;'
```

I tried to add all the extra escapes around `\` (to `\\`).

Reply:
You're right I rushed that a bit too quickly. I'll take a look at your suggestion and do some testing. 
The extra escapes always make it a bit more tricky in nextflow.

Reply:
My goal is to match only the last "." character in the string to replace with `.staramr.`. I think your suggestion would remove the file extension after the the last ".". What I am thinking of replacing it with is:
``` sed 's;\.(?!.*\.)$;.staramr.;' ``` or in nextflow as ``` sed 's;\\.(?!.*\\.)\$;.staramr.;' ```

Reply:
Actually I don't think it'll work if someone decides to name a file `text..txt`

Reply:
I tested it out on a number of different sample names
```bash
test.txt
testA.B.C.txt
test..txt
test.test.txt
test.csv
test.test.csv
testA.B.V.csv
test..csv
test.xlsx
tes.tes.xlsx
test.A.B.C.xlsx
test..xlsx
test.tsv
test.test.tsv
test.A.B.V.tsv
test..tsv
```
and within nextflow 
```nextflow run main.nf -profile test,docker --outdir ./results --input assets/samplesheet_copy.csv```

Looks good!

Reply:
Committed the change fa6f128